test/src/javascripts/toggle.spec.js

```diff
@@ -0,0 +1,76 @@
+var
```

Comment:
Could add an addition test, return the function without an `active_state` parameter, the assertion would be that the element has a class of active.